models/hh_cond_exp_traub.nestml

```diff
@@ -37,12 +37,12 @@ neuron hh_cond_exp_traub_neuron:
     V_m mV = E_L #  Membrane potential
 
     # equilibrium values for (in)activation variables
-    function alpha_n_init real = 0.032 * ( 15. - V_m ) / ( exp( ( 15. - V_m ) / 5. ) - 1. )
-    function beta_n_init real = 0.5 * exp( ( 10. - V_m ) / 40. )
-    function alpha_m_init real = 0.32 * ( 13. - V_m ) / ( exp( ( 13. - V_m ) / 4. ) - 1. )
-    function beta_m_init real = 0.28 * ( V_m - 40. ) / ( exp( ( V_m - 40. ) / 5. ) - 1. )
-    function alpha_h_init real = 0.128 * exp( ( 17. - V_m ) / 18. )
-    function beta_h_init real = 4. / ( 1. + exp( ( 40. - V_m ) / 5. ) )
+    function alpha_n_init real = 0.032 * ( 15. - V_m / 1 mV) / ( exp( ( 15. - V_m / 1 mV ) / 5. ) - 1. )
+    function beta_n_init real = 0.5 * exp( ( 10. - V_m / 1 mV) / 40. )
+    function alpha_m_init real = 0.32 * ( 13. - V_m / 1 mV) / ( exp( ( 13. - V_m / 1 mV) / 4. ) - 1. )
+    function beta_m_init real = 0.28 * ( V_m /1 mV  - 40. ) / ( exp( ( V_m / 1 mV - 40. ) / 5. ) - 1. )
+    function alpha_h_init real = 0.128 * exp( ( 17. - V_m / 1 mV) / 18. )
+    function beta_h_init real = 4. / ( 1. + exp( ( 40. - V_m / 1 mV ) / 5. ) )
```

Comment:
It should not be necessary to points everywhere (but I have to check what happens if it translated to C++). Yes you can use 15mV as an unit literal.